appcues/src/main/java/com/appcues/trait/appcues/SkippableTrait.kt

```diff
@@ -48,56 +63,78 @@ internal class SkippableTrait(
 ) : ContainerDecoratingTrait, BackdropDecoratingTrait {
 
     companion object {
-
         const val TYPE = "@appcues/skippable"
 
         private const val CONFIG_BUTTON_APPEARANCE = "buttonAppearance"
+        private const val CONFIG_BUTTON_STYLE = "buttonStyle"
         private const val CONFIG_IGNORE_BACKDROP_TAP = "ignoreBackdropTap"
-    }
 
-    private sealed class ButtonAppearance(val margin: Dp, val size: Dp, val padding: Dp) {
+        // default button is 30x30 - can override in buttonStyle
+        private const val BUTTON_DEFAULT_SIZE = 30.0
+
+        // default margin of 8 around every side - can override in buttonStyle
+        private const val BUTTON_DEFAULT_MARGIN = 8
+
+        // the 'X' drawn in the button will scale to 40% of button size (height)
+        private const val BUTTON_X_SCALE_FACTOR = 0.4
 
-        // calculates rippleRadius based on size and padding provided
-        val rippleRadius = (size / 2) - padding
+        // the 'X' in a minimal style button uses a 1.5 dp stroke width for a 30 dp size button
+        // and scales accordingly with different sizes
+        private const val BUTTON_X_MINIMAL_STROKE_FACTOR = 1.5
 
-        object Hidden : ButtonAppearance(0.dp, 0.dp, 0.dp)
-        object Minimal : ButtonAppearance(4.dp, 30.dp, 8.dp)
-        object Default : ButtonAppearance(8.dp, 30.dp, 8.dp)
+        // the 'X' in a default style button uses a 2.0 dp stroke width for a 30 dp size button
+        // and scales accordingly with different sizes
+        private const val BUTTON_X_DEFAULT_STOKE_FACTOR = 2.0
```

Comment:
these properties allow use to now scale with size of the X in a more consistent way with the symbol image / font size scaling used on iOS. The larger the button is, the more pronounced the X becomes. Also, on default, the X is a slightly heavier "weight" appearance, minimal it is thinner (2.0 vs 1.5 here)